.github/workflows/wait_for_required_workflows.yml

```diff
@@ -49,6 +49,7 @@ jobs:
                            "plugins/source/cloudflare",
                            "plugins/source/datadog",
                            "plugins/source/digitalocean",
+                           "plugins/source/facebookmarketing",
```

Comment:
This is not needed anymore, see https://github.com/cloudquery/cloudquery/pull/8805